CHANGELOG

```diff
@@ -0,0 +1,22 @@
+## What's Changed
+* Increase support for unions in incremental smt2 decision procedure by @thomasspriggs in https://github.com/diffblue/cbmc/pull/7926
+* Add ASV book CBMC examples by @peterschrammel in https://github.com/diffblue/cbmc/pull/7921
+* Add models for C library: getpwnam, getpwuid, and getopt by @tautschnig in https://github.com/diffblue/cbmc/pull/7919 and https://github.com/diffblue/cbmc/pull/7916
+* Shadow memory addresses now also support --pointer-check argument by @esteffin in https://github.com/diffblue/cbmc/pull/7936
+* [DOCS] Add documentation on CBMC Shadow Memory intrinsics by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7913
+* [DOCS] Improve doxygen for shadow_memory_util by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7930
+* [SYNTHESIZER] Allow goto-synthesizer accept all CBMC options  by @qinheping in https://github.com/diffblue/cbmc/pull/7900
+
+## Bug Fixes
+* Fix problem on array size L2 renaming by @esteffin in https://github.com/diffblue/cbmc/pull/7877
+* Fix shadow memory missing aggregation on non-compound bitvector types by @esteffin in https://github.com/diffblue/cbmc/pull/7935
+
+## Internal Improvements
+* Interval expressions: cleanup type hierarchy by @tautschnig in https://github.com/diffblue/cbmc/pull/7925
+* Rename {min,max}_exprt to {min,max}_value_exprt by @tautschnig in https://github.com/diffblue/cbmc/pull/7931
+* Introduce widen_if_needed by @kroening in https://github.com/diffblue/cbmc/pull/7938
+* format_expr: format constants with natural number type by @kroening in https://github.com/diffblue/cbmc/pull/7940
+* Enable all struct tests under `regression/cbmc/` for new SMT backend by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7809
```

Comment:
:pick: I suspect this is actually a misleadingly named PR, as we enabled the tests which pass only. There are at least some remaining to fix.